Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/events/server/qrcodepostertemplate/QrCodePosterTemplateCache.kt

```diff
@@ -0,0 +1,41 @@
+package de.rki.coronawarnapp.eventregistration.events.server.qrcodepostertemplate
+
+import android.content.Context
+import de.rki.coronawarnapp.util.di.AppContext
+import timber.log.Timber
+import java.io.File
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class QrCodePosterTemplateCache @Inject constructor(@AppContext private val context: Context) {
+
+    private val posterTemplateDir = File(context.filesDir, "events")
+
+    fun getTemplate(): ByteArray {
+        return File(posterTemplateDir, FILE_NAME).let { file ->
+            if (file.exists()) {
+                Timber.d("Loading template from $file")
+                file.readBytes()
+            } else {
+                Timber.d("Loading default template")
+                context.assets.open("default_qr_code_poster_template.pdf").readBytes()
```

Comment:
As the default poster, we have to use the binary of the Poster template **protobuf message**, not only the PDF itself (Similar to AppConfig). You can find it here: https://github.com/corona-warn-app/cwa-app-tech-spec/blob/proposal/event-registration-mvp/docs/spec/event-registration-client.md#protocol-buffer-message-qrcodepostertemplate

Reply:
The default template is from protobuf message, I just changed `.bin` to `.pdf` since it actually is pdf file,
but I can change it back to `.bin` and add `.sha256` to have the same format as in the app config

Reply:
Please consider having a default data as in here https://github.com/corona-warn-app/cwa-app-tech-spec/blob/a71ca36e09998a2ab92e764ade732e4546e30db7/docs/spec/event-registration-client.md#protocol-buffer-message-qrcodepostertemplate

Reply:
yes, the default template is from `default_qr_code_poster_template_android` in spec